server/midddleware/pprof.go

```diff
@@ -21,7 +21,7 @@ import (
 	"google.golang.org/grpc"
 )
 
-func PprofUnaryServerInterceptor() grpc.UnaryServerInterceptor {
+func pprofUnaryServerInterceptor() grpc.UnaryServerInterceptor {
```

Comment:
any reason of changing this?

Reply:
It's only used inside the the package, better to have only necessary scope.

Reply:
It is. See above:
```
-		PprofUnaryServerInterceptor(),
+		pprofUnaryServerInterceptor(),
```
It's just to signify that the function is not used outside of `middleware` package.

Reply:
that is fine but main point is then just change the other as well that are inside this package. Mainly - `TimeoutUnaryServerInterceptor`, `TxCtxUnaryServerInterceptor`

Reply:
Yes, this will be changed in the next diff.